exercises/custom-set/CustomSet.pm

```diff
@@ -0,0 +1,77 @@
+package CustomSet;
+use strict;
+use warnings;
+use Exporter 'import';
```

Comment:
We don't need the exporter for this as all the subs are class methods.

Reply:
Do we need `our @EXPORT_OK = ...` then?
```suggestion
```

Reply:
We don't need it. (Sorry for lack of clarity I meant to include `@EXPORT_OK` with the comment)